salt/utils/templates.py

```diff
@@ -345,11 +346,17 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
                                        **env_args)
 
     jinja_env.filters['strftime'] = salt.utils.date_format
-    jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
     jinja_env.filters['yaml_dquote'] = salt.utils.yamlencoding.yaml_dquote
     jinja_env.filters['yaml_squote'] = salt.utils.yamlencoding.yaml_squote
     jinja_env.filters['yaml_encode'] = salt.utils.yamlencoding.yaml_encode
 
+    for function_name in context['salt'].keys():
+        # Simply iterating over the keys forces the execution modules to load
```

Comment:
>Okay, so now it works also for execution modules thanks to that iteration. I don't see any for the utils.

No iteration loop is required for the utils modules because those functions are not dependent on the lazy loader. So, no, you won't see that.

>But I can see that you are doing the same mistake as previously, rewriting the same object. 

That is not a mistake. I did the reading you suggested on classes and decorators. It is not a new instance of the class. That is not how class attributes actually work. I did test this and can confirm that `JinjaFilter.salt_jinja_filters` is a dictionary that contains _all_ the decorated utils and execution module functions.